client/deis.py

```diff
@@ -1043,18 +1043,35 @@ def certs_add(self, args):
         """
         Binds a certificate/key pair to an application.
 
-        Usage: deis certs:add <cert> <key>
+        Usage: deis certs:add <cert> <key> [--subject-alt-name=<san>...] [options]
```

Comment:
I see what you mean here about `docopt` swallowing the options. This is a reasonable workaround.